Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -55,11 +55,8 @@ class TabLocationView: UIView {
             }
             updateTextWithURL()
             pageOptionsButton.isHidden = (url == nil)
```

Comment:
Since this logic uses same conditional, should be clumped into your statement. Maybe chain if no optionals:
`pageOptionsButton.isHidden = trackingProtectionButton.isHidden = (url == nil)

Reply:
I think the reason its like this is because they don't want to un-hide the tracking protection button based on a nil URL.

They update `trackingProtectionButton.isHidden` at [`updateBlockerStatus(forTab:)`](https://github.com/brave/brave-ios/blob/cf74fc019a64404bbcee1bd6162ef3932273f1c3/Client/Frontend/Browser/TabLocationView.swift#L385).

It's possible we can just remove this:

```swift
if url == nil {
    trackingProtectionButton.isHidden = true
}
```

Reply:
Just removed this. Not only do we not even use `trackingProtectionButton` because we don’t use their tracking protection, but it’s hidden status is updated elsewhere